test/basic-querying.test.js

```diff
@@ -958,6 +958,26 @@ describe('basic-querying', function() {
         null, // databases representing `undefined` as `null` (e.g. SQL)
       ]);
     });
+
+    describe('check __parent relationship in embedded models', () => {
+      it('should fill the parent in embedded model', async () => {
+        console.log();
+        const [user] = await User.find({where: {name: 'John Lennon'}});
```

Comment:
My guess is that the result of this line might be empty if the database uses `string` as the id type.

Reply:
Hi @agnes512.  Why would an `id` of **string** type affect the query since it searches using the `name` property which is already a **string**